cliquet/events.py

```diff
@@ -0,0 +1,15 @@
+from cliquet.utils import strip_uri_prefix
+
+
+class CollectionChanged(object):
+    """Triggered when a collection is changed.
+    """
+    def __init__(self, action, collection, request):
+        self.request = request
+
+        self.payload = {'collection_id': collection.record_id,
```

Comment:
Yes, what you suggested in the initial document was too tight to original http request IMO (verb, status code, etc.)